src/test_l3_dsFPD.c

```diff
@@ -0,0 +1,692 @@
+*
+* **Pre-Conditions:** @n
+* Front panel Module should be intialized through Test 1 before calling this test.
+* Front panel indicator state should be set to ON through Test 2 before calling this test
+*
+* **Dependencies:** None@n
+*
+* **User Interaction:** @n
+* User or Automation tool should select the Test 9 and provide indicator value and Color value.
+*
+*/
+void test_l3_dsFPD_hal_SetFPColor(void)
+{
+    gTestID = 10;
+    UT_LOG_INFO("In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
+    int32_t ret = 0;
+    dsError_t status = dsERR_NONE;
+    uint32_t uColor;
+    bool isValidColor = false;
+
+    ret = selectIndicator();
+    if(!ret)
+        goto exit;
+    UT_LOG_MENU_INFO(" \t  Supported Front Panel color for the indicator %s are:",UT_Control_GetMapString(dsFrontPanelIndicatorTable, eIndicator));
+    UT_LOG_MENU_INFO("------------------------------------------");
+
+    for (int32_t j = 0; j < dsFPD_COLOR_MAX; j++)
+    {
+        UT_LOG_MENU_INFO("\t0X%06X.  %-20s\n", dsFrontPanelColorTable[j].value, dsFrontPanelColorTable[j].string);
+    }
+    UT_LOG_MENU_INFO("----------------------------------------------------------");
+    UT_LOG_MENU_INFO("Select Color: ");
+    scanf("%u", &uColor);
+    readAndDiscardRestOfLine(stdin);
+    isValidColor = false;
+    for (int32_t j = 0; j < dsFPD_COLOR_MAX; j++)
+    {
+                if( dsFrontPanelColorTable[j].value == uColor)
+                    isValidColor = true;
+    }
+    if(!isValidColor)
+    {
+        UT_LOG_INFO("Unsupported Color Input 0x%08X",uColor);
+        goto exit;
+    }
+
+    /* Check that the Indicator is valid */
+    UT_LOG_INFO("Calling dsSetFPColor(IN:Indicator:[0x%d], IN:Color:[%X])" \
+                                                ,eIndicator,uColor);
+    status = dsSetFPColor((dsFPDIndicator_t)eIndicator,uColor);
+    UT_LOG_INFO("Result dsSetFPColor() dsError_t:[%s]",UT_Control_GetMapString(dsFrontPanelErrorCodeTable, status));
+    assert(status == dsERR_NONE);
+    exit:
+    UT_LOG_INFO("Out %s\n", __FUNCTION__);
+}
+/**
+* @brief This test provides a scope to get Color of the Indicator set.
+*
+* This test case provides a scope to  get the Color of Given indicator
+*
+* **Pre-Conditions:** @n
+* Front panel Module should be intialized through Test 1 before calling this test.
+* Front panel indicator state should be set to ON through Test 2 before calling this test
+*
+* **Dependencies:** None@n
+*
+* **User Interaction:** @n
+* User or Automation tool should select the Test 10 and provide indicator index.
+*
+*/
+void test_l3_dsFPD_hal_GetFPColor(void)
+{
+    gTestID = 11;
+    UT_LOG_INFO("In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
+
+    dsError_t status = dsERR_NONE;
+    dsFPDColor_t color;
+    int32_t ret = 0;
+
+    ret = selectIndicator();
+    if(!ret)
+        goto exit;
+    UT_LOG_INFO("Calling dsGetFPColor(IN:Indicator:[0x%d], OUT:Color)",eIndicator);
```

Comment:
make empty [] braces for the eg., OUT:Color[]

Reply:
added [] to all out param wherever requruired